### PR TITLE
Added Maximum of All Subarrays of Size K in Java

### PR DESCRIPTION
**Pull request by neha-b2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes, I have.

### Description

Added the **'Maximum of All Subarrays of Size K'** program in **Java**, solved using the 'Sliding Window Technique'.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2304 (Java) - mentioned under **'Fixed Window Problems'**

### CODE REVIEW

File: Java/cp/MaximumEachSubarray.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	    int len = Integer.parseInt(buf.readLine());
+        System.out.println("Enter array of integers: ");
+	    StringTokenizer st = new StringTokenizer(buf.readLine());
+	    int numArray[] = new int[len];
+	    for(int i=0; i<len; i++) {
+	        numArray[i] = Integer.parseInt(st.nextToken()); 
+	    }
+        System.out.println("Enter size of window: ");
+	    int K = Integer.parseInt(buf.readLine()); 
+
```
Comment (siddharth25pandey) [inline Java/cp/MaximumEachSubarray.java:67]: Display a small message what you are printing as output for better user-interface

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@neha-b2001 Minor change is required, Rest is fine.

File: Java/cp/MaximumEachSubarray.java (hunk 1 above)
Comment (neha-b2001) [reply Java/cp/MaximumEachSubarray.java:67]: @siddharth25pandey Okay :+1:

Comment (siddharth25pandey) [reply Java/cp/MaximumEachSubarray.java:67]: @neha-b2001 Please read carefully, In form of output not as a comment, same follows for your rest PR

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

File: Java/cp/MaximumEachSubarray.java (hunk 1 above)
Comment (neha-b2001) [reply Java/cp/MaximumEachSubarray.java:67]: @siddharth25pandey Sorry, I'll fix this right away. 
Done :+1:

**Review by siddharth25pandey — APPROVED** [review]
@neha-b2001 LGTM, able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @neha-b2001 for her code

**Pull request merged**